### PR TITLE
Update travis configuration to use container based infrastructure, al…

### PR DESCRIPTION
**Pull request by henrikbjorn**
…so use cache for composer in order to reduce network usage

Also reformatted yml to use the standard 2 space indentation.

### CODE REVIEW

**Comment by fabpot** [comment]
We are using 4 spaces everywhere else.

**Comment by henrikbjorn** [comment]
#1226 fixes the hhvm test that fails.

**Comment by henrikbjorn** [comment]
4 spaces have been reintroduced in the travis file.

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
-
 php:
-  - 5.5
-  - 5.6
-  - 7.0
-  - hhvm
+    - 5.5
+    - 5.6
+    - 7.0
+    - hhvm
```
Comment (fabpot) [inline .travis.yml]: Can you remove this `php` section and move those version of PHP in the matrix? That simplifies the file.

Comment (fabpot) [inline .travis.yml]: Can you rebase on current master and remove `hhvm` from the allowed failures?

**Comment by henrikbjorn** [comment]
@fabpot rebased against master and added the php section into the matrix.

**Comment by fabpot** [comment]
That's something we need to do on the 1.3 branch instead.

**Comment by fabpot** [comment]
closing in favor of #1227

**Pull request closed without merging**